src/components/Search/Search.js

```diff
@@ -64,10 +71,28 @@ function Search({
     [apiKey, engines, setSuggestions],
   );
 
+  const theme = {
```

Comment:
Maybe we should put this into a `useMemo` hook? Might be a small performance improvement.

Reply:
Yes it is better so :)